docs/website/content/v0.3/en/configuration/v1alpha1.md

```diff
@@ -331,6 +331,37 @@ sysctls:
 
 ```
 
+#### registries
+
+Used to configure the machine's container image registry mirrors.
+Automatically generates matching CRI configuration for registry mirrors.
+
+Type: `RegistriesConfig`
+
+Examples:
+
+```yaml
+registries:
+  mirrors:
+    docker.io:
+      endpoints:
+        - https://registry-1.docker.io
+    '*':
+      endpoints:
+        - http://some.host:123/
+ config:
+  "some.host:123":
+    tls:
+      CA: ...
+      clientIdentity:
+        cert: ...
+        key: ...
+    auth:
```

Comment:
this block looks like it needs to be indented by one more space.

Reply:
looks like it's correct, `auth` & `tls` are on the same level, unless I'm missing something